tests/test-http-post-1mb.lua

```diff
@@ -42,9 +42,9 @@ require('tap')(function(test)
       request:on('end', expect(function()
         p('server:onEnd')
         assert(postBuffer == data)
-        response:write(data)
```

Comment:
Moving server:close() seems ok but we shouldn't need to wrap the response:finish() in the callback.